packages/admin/admin/src/mui/menu/CollapsibleItem.tsx

```diff
@@ -1,96 +1,192 @@
+                onMouseOver={handlePopoverOpen}
+                onMouseLeave={handlePopoverClose}
+            >
                 <MenuItem
                     primary={primary}
                     secondary={secondary}
                     hasSubitems={!!childElements?.length}
-                    isCollapsibleOpen={open}
+                    isCollapsibleOpen={isSubmenuOpen}
+                    isMenuOpen={isMenuOpen}
                     icon={icon}
                     level={level}
-                    onClick={() => setOpen(!open)}
-                    secondaryAction={open ? openedIcon : closedIcon}
+                    onClick={() => setIsSubmenuOpen(!isSubmenuOpen)}
+                    secondaryAction={
+                        itemLevel === 1 && isSubmenuOpen ? (
+                            <OpenedIcon
+                                className={`${classes.collapsibleIcon} ${
+                                    !isMenuOpen && isSubmenuOpen && itemLevel === 1 ? classes.colorWhite : classes.colorGrey
+                                }`}
+                            />
+                        ) : (
+                            <ClosedIcon
+                                className={`${classes.collapsibleIcon} ${
+                                    !isMenuOpen && isSubmenuOpen && itemLevel === 1 ? classes.colorWhite : classes.colorGrey
+                                }`}
+                            />
+                        )
+                    }
+                    className={listClasses.join(" ")}
                 />
             </div>
-            <Collapse in={open} timeout="auto" unmountOnExit>
-                <List disablePadding>{childElements}</List>
-            </Collapse>
+            {!isMenuOpen && drawerVariant === "permanent" ? (
+                <Menu
+                    id="mouse-over-menu"
+                    MenuListProps={{
+                        "aria-labelledby": "menu-item",
+                        onMouseLeave: closeMenu,
+                        style: { pointerEvents: "auto" },
+                    }}
+                    sx={{
+                        pointerEvents: "none",
+                    }}
+                    open={isSubmenuOpen}
+                    anchorEl={anchorEl}
+                    TransitionComponent={Fade}
+                    anchorOrigin={{
+                        vertical: "center",
+                        horizontal: "right",
+                    }}
+                    transformOrigin={{
+                        vertical: "center",
+                        horizontal: "left",
+                    }}
+                    onClose={handlePopoverClose}
+                >
+                    <List disablePadding>
+                        {itemLevel === 1 && (
+                            <Typography className={classes.itemTitle} variant="h3">
+                                {primary}
+                            </Typography>
+                        )}
```

Comment:
Should the title be rendered before the list, instead of inside it, as it's not actually a list-item?

Reply:
✅